docs/getting-started/getting-started-with-notebooks.rst

```diff
@@ -137,26 +133,3 @@ When viewing a public notebook on Dockstore, you can launch the notebook into Go
 
 .. figure:: /assets/images/docs/launch-with-google-colab.png
 
-Using the Notebooks Feature Flag
---------------------------------
-
-There is a ``notebooks`` feature flag that can be used to explore Dockstore Notebooks features that are in development.
-
-.. warning:: The features hidden behind this flag may not be fully functional, but using the flag offers you a preview of the features to come.
-
-To use the ``notebooks`` feature flag, append ``notebooks`` to the Dockstore URL as a query parameter. You only need to do this once, unless you refresh/close your browser.
-
-For example, if you're on the https://dockstore.org page, append ``notebooks`` such that it looks like this: https://dockstore.org?notebooks.
-
-If you're on a page that already contains query parameters, indicated by the presence of a question mark, append ``notebooks`` to the URL using an ampersand. For example, if you're on the https://dockstore.org/starred?tab=workflows page, append ``notebooks`` such that it looks like https://dockstore.org/starred?tab=workflows&notebooks.
-
-After applying the notebooks feature flag, Dockstore Notebooks features will appear throughout the site, marked with a warning banner.
```

Comment:
Some of the images on this page display the warning banner. Is it worth updating these screenshots?

Reply:
Yes, I will do it in the next PR, which should be up for review today or tomorrow.